src/jit/jitconfigvalues.h

```diff
@@ -218,6 +218,10 @@ CONFIG_INTEGER(JitEnableNoWayAssert, W("JitEnableNoWayAssert"), 0)
 CONFIG_INTEGER(JitEnableNoWayAssert, W("JitEnableNoWayAssert"), 1)
 #endif // !defined(DEBUG) && !defined(_DEBUG)
 
+#if defined(_TARGET_AMD64_) && defined(FEATURE_CORECLR)
+CONFIG_INTEGER(JitMinOptsTrackGCrefs, W("JitMinOptsTrackGCrefs"), 0) // Allow GC roots to be tracked w/MinOpts
```

Comment:
Yes, I was suggesting eliminating all the `#ifdefs`, so it would be possible to enable and test on desktop, or on non-AMD64 platforms, simply by setting a COMPlus variable. We have very few `#ifdef FEATURE_CORECLR` and I'd like to keep it that way.